testing/integration-tests/proc-macro/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "integration-tests-proc-macro"
```

Comment:
Nit: I'd call this crate something like `subxt-test-proc-macro` I think, so it's super easy to relate it to the `#[subxt_test]` attr it exports